manifests/h/HERE/OLPCLI/0.0.0/HERE.OLPCLI.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: HERE.OLPCLI
+PackageVersion: 0.0.0

```

```suggestion
PackageVersion: 10.5.106
```
This must match Control Panel.